### ISSUE TITLE
PDF 1.7 merge not working

### ISSUE THREAD

**Issue by applyACS** [hanneskod/libmergepdf#49]
When merging with PDF 1.7 version it is not working
"'Array to string conversion' in 'raw-content'" - I get this when using TcpdiDriver
and of course:
"'This PDF document probably uses a compression technique which is not supported by the free parser shipped with FPDI. (See https://www.setasign.com/fpdi-pdf-parser for more details)' in 'raw-content'" - this for FPDI

**Comment by hanneskod** [comment]
Yes some pdf features seems to make tcpdi blow up. I need the pdfs you are trying to merge to be able to reproduce. What php and lib versions are you using?

**Comment by applyACS** [comment]
[1582646396_phpXTaAZU.pdf](https://github.com/hanneskod/libmergepdf/files/4750182/1582646396_phpXTaAZU.pdf)
This is the pdf 1.7. The other one is generated using wkhtmltopdf(no issue here).
php 7.3 or 7.4
iio/libmergepdf: 4.0.3

**Comment by lgme** [comment]
I encounter the same issues as above. Do you have any results @hanneskod ?

**Comment by kaystrobach** [comment]
@lgme can you please also provide the file - this makes it easier to debug and maybe raise a bugreport at Tcpdi and other used projects.

**Comment by lgme** [comment]
@kaystrobach Sure. Here:
[p1.pdf](https://github.com/hanneskod/libmergepdf/files/5500174/p1.pdf)
[p2.pdf](https://github.com/hanneskod/libmergepdf/files/5500175/p2.pdf)

**Comment by kaystrobach** [comment]
awesome! - will try to check that in the next days.

**Comment by kaystrobach** [comment]
sry took longer than expected, will try to take a look during the holidays

**Comment by pulponair** [comment]
Any news?

**Comment by OlexandrPopov** [comment]
It seems that for PDF 1.7 you need a commercial FPDI PDF-Parser. See more details [here](https://www.setasign.com/support/faq/fpdi/error-document-compression-technique-not-supported/#question-89) and [here](https://github.com/Setasign/FPDI/issues/32).
